### PR TITLE
Fix `blitz db` commands showing incorrect error (patch)

### PR DESCRIPTION
**Pull request by arjundubey-cr**
Closes: #1579 

### What are the changes and their implications?
updated db.ts to provide more specific error.
### Checklist

- [ ] Tests added for changes
- [ ] PR submitted to [blitzjs.com](https://github.com/blitz-js/blitzjs.com) for any user facing changes

### CODE REVIEW

File: packages/cli/src/commands/db.ts [hunk 1]
```diff
@@ -9,10 +9,8 @@ const runPrisma = async (args: string[], silent = false) => {
   if (!prismaBin) {
     try {
       prismaBin = await getPrismaBin()
-    } catch {
-      throw new Error(
-        "Oops, we can't find Prisma Client. Please make sure it's installed in your project",
-      )
+    } catch (err) {
+      log.error(err)
```
Comment (KATT) [inline packages/cli/src/commands/db.ts:13]: you still probably wanna throw an error

Comment (arjundubey-cr) [reply packages/cli/src/commands/db.ts:13]: Ok I'll do that

Comment (arjundubey-cr) [reply packages/cli/src/commands/db.ts:13]: I've updated the code

File: packages/cli/src/commands/db.ts [hunk 2]
```diff
@@ -9,7 +9,8 @@ const runPrisma = async (args: string[], silent = false) => {
   if (!prismaBin) {
     try {
       prismaBin = await getPrismaBin()
-    } catch {
+    } catch (err) {
+      log.error(err)
       throw new Error(
         "Oops, we can't find Prisma Client. Please make sure it's installed in your project",
       )
```
Comment (flybayer) [inline packages/cli/src/commands/db.ts:16]: Thanks! An even better solution is just to throw the original error which fixes the original issue. And this code is going to be removed soon anyways, so not too big of a deal.

```suggestion
      throw err
```

**Review by flybayer — APPROVED** [review]

**Pull request merged**